### PR TITLE
raysession: Fix issues with qt not being patched (can’t find platform, qt version mismatch error etc)

### LINKED ISSUE #228223: Bug - RaySession fails to start
### Describe the bug
The new package for [raysession](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/audio/raysession/default.nix) fails to start due to an error involving Qt. I have reproduced this behavior on two different computers.

### Steps To Reproduce
1. Add raysession to system configuration
2. Open raysession

Raysession does not open, and produces an error in the terminal.

### The Error

```
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

zsh: IOT instruction (core dumped)  raysession
```

### Notify maintainers

@orivej

### Metadata
Output of `nix-shell -p nix-info --run "nix-info -m"`.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.25, NixOS, 23.05 (Stoat), 23.05.20230422.6c43a34`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(user): `""`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by tobiasBora**
###### Description of changes

Fix https://github.com/NixOS/nixpkgs/issues/228223

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by PowerUser64 — APPROVED** [review]
LGTM! Tested with JACK and everything I tried works.

**Comment by PowerUser64** [comment]
Just tested with `nixpkgs-review pr 236762` and everything passed.

File: pkgs/applications/audio/raysession/default.nix [hunk 1]
```diff
@@ -35,7 +36,10 @@ buildPythonApplication rec {
   ];
 
   postFixup = ''
-    wrapPythonProgramsIn "$out/share/raysession/src" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/share/raysession/src" "$out $pythonPath" "$out/bin"
```
Comment (natsukium) [inline pkgs/applications/audio/raysession/default.nix:39]:
```suggestion
    wrapPythonProgramsIn "$out/share/raysession/src" "$out $pythonPath"
```
It seems that this parameter is unnecessary. 
https://github.com/NixOS/nixpkgs/blob/58e6faecd51ea396019419764eaf43ba993a8ba7/pkgs/development/interpreters/python/wrap.sh#L42-L45

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -11652,9 +11652,7 @@ with pkgs;
 
   pystring = callPackage ../development/libraries/pystring { };
 
-  raysession = python3Packages.callPackage ../applications/audio/raysession {
-    inherit (qt5) qttools;
-  };
+  raysession = python3Packages.callPackage ../applications/audio/raysession {};
```
Comment (natsukium) [inline pkgs/top-level/all-packages.nix:11655]:
```suggestion
  raysession = python3Packages.callPackage ../applications/audio/raysession {
    inherit (qt5) qttools wrapQtAppsHook;
  };
```
I think it is a good idea to add `wrapQtAppsHook` here.

Comment (tobiasBora) [reply pkgs/top-level/all-packages.nix:11655]: I don't get it: what's the advantage of this compared to doing `qt5.wrapQtAppHook` as I do now? For me it is just harder to test as I need to remember to supply to good arguments to callPackage, and I only saw a single other package (by the same author) importing qttools this way. Am I missing something?

Comment (natsukium) [reply pkgs/top-level/all-packages.nix:11655]: Not only `qttools`, but there are other examples like `qmake`, `qtbase`, `wrapQtAppsHook`, etc. It is also commonly done in other namespaces, not just `qt5`.
The advantage is that you can separate qt version changes from application changes. For example, `qutebrowser`.

On the other hand, it does not seem to be that much of an advantage for this application, but I do not think it is necessary to change it actively.

Comment (Artturin) [reply pkgs/top-level/all-packages.nix:11655]: https://github.com/NixOS/nixpkgs/issues/204303

**Pull request merged**